core/src/commonMain/kotlin/entity/User.kt

```diff
@@ -37,6 +37,11 @@ public open class User(
     override val id: Snowflake
         get() = data.id
 
+    /**
+     * Returns true if the user is the same as the bot.
+     */
+   public val isSelf: Boolean get() = id == kord.selfId
```

Comment:
```suggestion
    public val isSelf: Boolean get() = id == kord.selfId
```

Reply:
Might fix it

Reply:
We commited the same thing at the same time - mind if I just resolve that?

Reply:
Did you break it again?

Reply:
yeah just resolve it, It looks fine in the changes window